solutions/cron/pages/index.tsx

```diff
@@ -0,0 +1,100 @@
+import { Layout, Text, Code, Page, Link } from '@vercel/examples-ui'
+import Post from '@/components/post'
+import Head from 'next/head'
+
+const intervals = [
+  {
+    id: '1m',
+    name: 'Every Minute',
+    cron: '* * * * *',
+  },
+  {
+    id: '10m',
+    name: 'Every 10 Minutes',
+    cron: '*/10 * * * *',
+  },
+  {
+    id: '1h',
+    name: 'Every Hour',
+    cron: '0 * * * *',
+  },
+  {
+    id: '6h',
+    name: 'Every 6 Hours',
+    cron: '0 */6 * * *',
+  },
+  {
+    id: '1d',
+    name: 'Every Day',
+    cron: '0 0 * * *',
+  },
+]
+
+const image = 'https://cron-template.vercel.app/thumbnail.png'
+
+export default function Home({ data }: { data: any }) {
```

Comment:
Could we render a warning if we detect that EDGE_CONFIG is not defined? You can likely copy it from the feature-flags-apple-store or maintenance-page examples.